### ISSUE TITLE
status_check throws error

### ISSUE THREAD

**Issue by logdot** [infobyte/faraday#281]
## Faraday version
Paste the output of the *./faraday.py --version* command
Faraday v3.0.1

## Component Name
server/commands/status-check.py

## Steps to reproduce
When running manage.py with the status_check flag it throws the error

```
Traceback (most recent call last):
  File "manage.py", line 199, in <module>
    cli()
  File "/usr/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "manage.py", line 133, in status_check
    status_check_functions.full_status_check()
  File "/root/faraday-dev/server/commands/status_check.py", line 295, in full_status_check
    print_faraday_status()
  File "/root/faraday-dev/server/commands/status_check.py", line 212, in print_faraday_status
    pid = check_server_running()
  File "/root/faraday-dev/server/commands/status_check.py", line 31, in check_server_running
    pid = is_server_running()
TypeError: is_server_running() takes exactly 1 argument (0 given)
```

## Expected results
To get this output
```
Checking if postgreSQL is running...
[+] PostgreSQL is running
[+] PostgreSQL lock not detected. 

Checking if Faraday is running...
[+] Faraday Server is running. PID:2312         
[-] Faraday GTK is not running

Checking Faraday dependencies...
[+] Server dependencies met
[+] Client dependencies met

Checking Faraday config...
[-] Error. Credentials does not match
[+] /.faraday/storage -> Permission accepted
[+] Port 5985 in 0.0.0.0 is open
```
It fails at
` [+] Faraday Server is running. PID:2312   

### Debugging tracebacks (current results)
No logs

Pip freeze:
```
appdirs==1.4.3
aurman==2.18
CacheControl==0.12.5
chardet==3.0.4
click==6.7
colorama==0.3.9
configparser==3.5.0
distlib==0.2.7
distro==1.3.0
feedparser==5.2.1
html5lib==1.0.1
idna==2.7
lockfile==0.12.2
lxml==4.2.5
Markdown==2.6.11
msgpack==0.5.6
packaging==17.1
progress==1.4
psycopg2-binary==2.7.5
pyalpm==0.8
Pygments==2.2.0
pyparsing==2.2.1
python-dateutil==2.7.3
pytoml==0.1.16
regex==2018.8.17
requests==2.19.1
retrying==1.3.3
six==1.11.0
urllib3==1.23
webencodings==0.5.1
```

### Screenshots
https://drive.google.com/file/d/10hvoL5A3Ox8rnLZRQZnXH6LUEceHOI2u/view?usp=sharing

### Configuration files

No changes

### OS
- Kernel: x86_64 Linux 4.18.3-arch1-1-ARCH

uname -a
` Linux uranium 4.18.3-arch1-1-ARCH #1 SMP PREEMPT Sat Aug 18 09:22:54 UTC 2018 x86_64 GNU/Linux

**Comment by Ezequieltbh** [comment]
To be fixed in version 3.1.1
Thanks for report this!

**Comment by Ezequieltbh** [comment]
Fix deployed in the new version.
Thanks for contribute to Faraday!
